endorsement/dao/shared_drive.py

```diff
@@ -1,65 +1,99 @@
 # Copyright 2024 UW-IT, University of Washington
 # SPDX-License-Identifier: Apache-2.0
 
+import collections
+import datetime as dt
+from endorsement.dao.gws import is_group_member
+from endorsement.dao.itbill import get_subscription_by_key_remote
+from endorsement.exceptions import ITBillSubscriptionNotFound
+from endorsement.models import (
+    ITBillSubscription,
+)
+
 from endorsement.models.shared_drive import (
-    SharedDriveMember, Member, Role, SharedDrive,
-    SharedDriveQuota, SharedDriveRecord)
+    Member,
+    Role,
+    SharedDrive,
+    SharedDriveMember,
+    SharedDriveQuota,
+    SharedDriveRecord,
+)
 from endorsement.exceptions import SharedDriveNonPrivilegedMember
-import csv
 import re
 import logging
 
+# methods are imported to populate namespace and be used elsewhere
+from uw_msca.shared_drive import (  # noqa: F401
+    get_default_quota,
+    get_google_drive_states,
+    set_drive_quota,
+)
+from uw_msca.models import GoogleDriveState
+
 
 logger = logging.getLogger(__name__)
 netid_regex = re.compile(
-    r'^(?P<netid>[^@]+)@(uw|(u\.)?washington)\.edu$', re.I)
+    r"^(?P<netid>[^@]+)@(uw|(u\.)?washington)\.edu$", re.I
+)
 
 
-def load_shared_drives_from_csv(file_path):
+def load_shared_drives(google_drive_states):
```

Comment:
Notable change: load_shared_drives management command is now loading from the MSCA endpoint and not a CSV file